### DISCUSSION TITLE
Will Nue JS support non-JS backends?

### DISCUSSION THREAD

**Discussion by JensRoland** [nuejs/nue#33] [Q&A] [answered]
Exciting project!!! If I understand correctly, with NueKit we will have a backend component to the Nue JS frontend. Will it be possible to build backends in other languages if you are less enamored with JS-on-the-server (even with Bun finally making this much more bearable)?

Would it be possible for the PHP/Laravel community, the Python/Django community, the Go/Gin/Fiber community etc. to develop their own server 'adapters' to work with Nue JS, or will the toolchain be JS end to end?

**Comment by tipiirai** [comment, marked as answer]
Great to hear! Nue JS is not tied to any specific backend. It's like React or Vue — a frontend library to build user interfaces. It's just the view part which can communicate with any backend using the native `fetch()` command. PHP or Django communities can start adopting it right away (remembering that it's still v 0.1 :)

Nuekit actually a static website generator built on top of Bun (or Node) that uses all the different tools on the Nue toolset.
